R/estimate_infections.R

```diff
@@ -189,7 +202,7 @@ estimate_infections <- function(reported_cases,
 
   # Define stan model parameters
   data <- create_stan_data(
```

Comment:
```suggestion
  stan_data <- create_stan_data(
```
and then will have to refer to `stan_data` in lines 215 and 224.

Reply:
Thanks. It makes sense to change all the objects named `data` to `stan_data` as it's more descriptive, so I made the change in all 3 models. See 5d279e8e2240f6491b2d4cdc8231087d8b5c7ac1.